cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -10410,3 +10410,73 @@ struct_init.cpp:
 #   36|     v36_9(void)           = ReturnVoid                       : 
 #   36|     v36_10(void)          = AliasedUse                       : ~m?
 #   36|     v36_11(void)          = ExitFunction                     : 
+
+try_except.c:
+#    6| void f()
+#    6|   Block 0
+#    6|     v6_1(void)            = EnterFunction                  : 
+#    6|     mu6_2(unknown)        = AliasedDefinition              : 
+#    6|     mu6_3(unknown)        = InitializeNonLocal             : 
+#    7|     r7_1(glval<int>)      = VariableAddress[x]             : 
+#    7|     mu7_2(int)            = Uninitialized[x]               : &:r7_1
+#    7|     r7_3(glval<int>)      = VariableAddress[y]             : 
+#    7|     r7_4(int)             = Constant[0]                    : 
+#    7|     mu7_5(int)            = Store[y]                       : &:r7_3, r7_4
+#    9|     r9_1(glval<unknown>)  = FunctionAddress[ProbeFunction] : 
+#    9|     r9_2(int)             = Constant[0]                    : 
+#    9|     v9_3(void)            = Call[ProbeFunction]            : func:r9_1, 0:r9_2
+#    9|     mu9_4(unknown)        = ^CallSideEffect                : ~m?
+#   10|     r10_1(glval<int>)     = VariableAddress[y]             : 
+#   10|     r10_2(int)            = Load[y]                        : &:r10_1, ~m?
+#   10|     r10_3(glval<int>)     = VariableAddress[x]             : 
+#   10|     mu10_4(int)           = Store[x]                       : &:r10_3, r10_2
+#   11|     r11_1(glval<unknown>) = FunctionAddress[ProbeFunction] : 
+#   11|     r11_2(int)            = Constant[0]                    : 
+#   11|     v11_3(void)           = Call[ProbeFunction]            : func:r11_1, 0:r11_2
+#   11|     mu11_4(unknown)       = ^CallSideEffect                : ~m?
+#   16|     v16_1(void)           = NoOp                           : 
+#    6|     v6_4(void)            = ReturnVoid                     : 
+#    6|     v6_5(void)            = AliasedUse                     : ~m?
+#    6|     v6_6(void)            = ExitFunction                   : 
+
+#   13|   Block 1
+#   13|     r13_1(int) = Constant[0] : 
+
+#   14|   Block 2
+#   14|     r14_1(glval<unknown>) = FunctionAddress[sink] : 
+#   14|     r14_2(glval<int>)     = VariableAddress[x]    : 
+#   14|     r14_3(int)            = Load[x]               : &:r14_2, ~m?
+#   14|     v14_4(void)           = Call[sink]            : func:r14_1, 0:r14_3
+#   14|     mu14_5(unknown)       = ^CallSideEffect       : ~m?
```

Comment:
I expect an `ExitFunction` instruction to be reached from here.

Reply:
Hm, yes. That's odd. Let me take a look at that 👀.

Reply:
Ah, yes. I messed up the catch handling. I'll push a fix before lunch today 🤞.

Reply:
Fixed in e6a03a6152755ef803175a05bea9ea725d26847c.